src/translib/transformer/sw_vlan.go

```diff
@@ -1203,6 +1203,14 @@ var YangToDb_sw_vlans_xfmr SubTreeXfmrYangToDb = func(inParams XfmrParams) (map[
     if err != nil {
         return nil, err
     }
+    /* Restrict configuring member-port if Physical interface configured as lag interface */
+    if intfType == IntfTypeEthernet {
+        err = validateIntfAssociatedWithPortChannel(inParams.d, &ifName)
+        if err != nil {
+            errStr := "VLAN config is not permitted on LAG member port"
```

Comment:
It would be better if we could point the LAG associated with it. To be clear, better to give meaningful error message.
Say: Port-Channel: 20 configuration exists on Interface: Ethernet24

Reply:
agreed , will do, thanks.